examples/determinator-rails/config/initializers/determinator.rb

```diff
@@ -3,5 +3,6 @@
 Determinator.configure(
   retrieval: Determinator::Retrieve::Routemaster.new(
     discovery_url: 'https://florence.dev/'
+    retrieval_cache: ActiveSupport::Cache::MemoryStore.new(expires_in: 1.minute)
```

Comment:
Might want to add `ActiveSupport` as an explicit dependency in the gemspec (it's already in `routemaster-drain`, but just in case that gets removed!)

Reply:
…or having `NullCache` as the default?

Reply:
This is example code, right? There's nothing stopping the user from using an alternative cache implementation.